src/Wpf.Ui/Controls/NavigationView/NavigationCache.cs

```diff
@@ -22,16 +22,34 @@ internal class NavigationCache
 
         if (cacheMode == NavigationCacheMode.Disabled)
         {
+#if DEBUG
+            System
+                .Diagnostics
+                .Debug
+                .WriteLine($"Cache for {entryType} is disabled. Generating instance using action...");
```

Comment:
Btw, there is no need in #DEBUG, because `Debug` [methods has a special attribute](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.conditionalattribute?view=net-8.0)